server/plugin/command.go

```diff
@@ -152,7 +152,7 @@ func (p *Plugin) handleMuteAdd(args *model.CommandArgs, username string, userInf
 	if err := p.API.KVSet(userInfo.UserID+"-muted-users", []byte(mutedUsers)); err != nil {
 		return "Error occurred saving list of muted users"
 	}
-	return fmt.Sprintf("`%v`", username) + " is now muted"
+	return fmt.Sprintf("`%v`", username) + " is now muted. You will not longer receive notifications for comments in your PRs and issues."
```

Comment:
Nit: Think this should say "You will no longer" rather than "not longer

Reply:
Sorry, I don't get why the issue is. Could you please post your suggestion?